src/components/RaasProviderCard.tsx

```diff
@@ -0,0 +1,33 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
+export const RaasProviderCard = ({
+  name,
+  description,
+  caption,
+  link,
+  logo,
+}: {
+  name: string;
+  description: string;
+  caption: string;
+  link: string;
+  logo: string;
+}) => {
+  return (
+    <Link href={link}>
```

Comment:
We should loop in Val and audit the design for both of these similar pages and come up with 1 final RaaS provider card design.
https://portal.arbitrum.io/orbit/launch